auto.def

```diff
@@ -902,8 +884,8 @@ foreach dir $subdirs {
 ###############################################################################
 # Provide timestamps (UTC) based on most recent author date or PACKAGE_VERSION
 # doc/neomuttrc.man, doc/neomutt.1
```

Comment:
I would also change the comment to something like:

Define package timestamp (UTC) based on PACKAGE_VERSION for:
doc/neomuttrc.man, doc/neomutt.1